### ISSUE TITLE
path not in POST

### ISSUE THREAD

**Issue by helge42** [gregplaysguitar/django-fileman#4]
Hi,
I'm testing your fileman-app and discovered that the delete and rename is not working.
The last step in the process is a POST-action, but in the view.delete the first line is
`path = request.GET.get('path')`
so the variable path is empty.
A quick fix for delete is:

```
def delete(request):
    if request.method == 'POST':
        path = request.POST.get('path')
        is_dir = os.path.isdir(get_full_path(path))
        if is_dir:
            shutil.rmtree(get_full_path(path))
        else:
            os.remove(get_full_path(path))

        msg = '%s was deleted' % path
        messages.add_message(request, messages.INFO, msg)
        return redirect('admin:fileman_upload_changelist')

    path = request.GET.get('path')
    is_dir = os.path.isdir(get_full_path(path))

    if not is_safe_path(path):
        return redirect('admin:fileman_upload_changelist')


    if is_dir:
        delete_results = render_delete_results(read_dir(path, True))
    else:
        delete_results = None

    return render(request, 'fileman/delete.html', {
        'app_label': 'fileman',
        'title': 'Delete',
        'delete_results': delete_results,
        'path': path,
    })

```
An in the template delete.html:

```
  <form action="{{ request.path }}" method="POST">
    {% csrf_token %}
    <input type="submit" value="Yes, I'm sure" class="delete-button">
    <input type="hidden" name="path" value="{{ path }}">
    <a class="button cancel-link" onclick="window.history.back(); return false;">
      No, take me back</a>
  </form>

```
Except of these it is nice app,
Regards, Helge

**Comment by gregplaysguitar** [comment]
Hi, yes sorry it is a bit of a work in progress at this stage - if you could package these changes into a pull request I'll merge them in.

Thanks!
Greg

**Comment by gregplaysguitar** [comment]
Ok, this should be fixed in v1.0.2 - can you try updating and let me know?

**Comment by helge42** [comment]
Yes, Version 1.0.2 is fixed! Great!
